controller.go

```diff
@@ -192,12 +195,12 @@ func New(cfgOptions ...config.Option) (NetworkController, error) {
 			dcfg = c.makeDriverConfig(i.ntype)
 		}
 
-		if err := drvRegistry.AddDriver(i.ntype, i.fn, dcfg); err != nil {
+		if err := drvRegistry.AddDriver(i.ntype, i.fn, dcfg, c.plugingetter); err != nil {
```

Comment:
If you store the plugin getter in the `drvRegistry`, `AddDriver()` does prototype does not need to change